README.md

```diff
@@ -90,6 +101,10 @@ git commit -am "I still do!"            # A combination of the above
 git commit --amend                      # Re-do the commit message of the previous commit (don't do this after pushing!)
                                         #   We _never_ change "public history"
 
+git reset <commit_hash>                 # Undo all commits after <commit_hash> (local changes are preserved)
```

Comment:
This is not actually a correct description of reset. And providing a correct one is out-of-scope for this README in my opinion.

Reply:
Ok, deleted.